common/src/test/scala/org/neo4j/spark/util/DummyNamedReference.scala

```diff
@@ -0,0 +1,7 @@
+package org.neo4j.spark.util
+
+import org.apache.spark.sql.connector.expressions.NamedReference
+
+abstract class DummyNamedReference extends NamedReference {
```

Comment:
maybe we could make it a concrete class with a constructor that takes a list of strings and `describe` just joins the list into a string? that would make the tests much less verbose. WDYT?

Reply:
yes I did as is as was easier to do a find/replace with just the name of the class, but I can do it in that way

Reply:
done